src/protagonist/API.Tests/Integration/ModifyAssetTests.cs

```diff
@@ -540,7 +541,37 @@ public async Task Put_New_Asset_Requires_Origin()
         // assert
         response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
     }
+    
+    [Theory]
+    [InlineData("deliveryChannels")]
+    [InlineData("wcDeliveryChannels")]
+    public async Task Put_New_Asset_Supports_DeliveryChannels_Aliases(string deliveryChannelAlias)
+    {
+        var assetId = new AssetId(99, 1, nameof(Put_New_Asset_Supports_DeliveryChannels_Aliases));
```

Comment:
The assetId here will need to be different across the requests, or it will be a Create then Update. Which _could_ give the illusion that it's fine when it could actually have created with first then ignored the second.

```cs
var assetId = new AssetId(99, 1, $"{nameof(Put_New_Asset_Supports_DeliveryChannels_Aliases)}-{deliveryChannelAlias}");
```

Reply:
Good point, I'll include this